releases.md

```diff
@@ -0,0 +1,103 @@
+# Tekton Release Cycles
+
+## Support Policy
+
+The Tekton project maintains three release branches for each project, created one per quarter, which 
```

Comment:
Doesn't this mean 9 months of support?

Reply:
Yes, good catch I mixed words up - I meant once every 4 month months, not every quarter (which is 3 months) 🤦 
I will update it.

Reply:
Uhm, I think I was also confused by [k8s release page](https://kubernetes.io/releases/) that says that k8s provides support for approximately one year,  it releases every three months and maintains branches for the most recent three minor releases, which doesn't add up, as it would be nine months.

It sounds like we do need quarterly releases to align with k8s, but we would have to keep four branches open rather than three, so provide one year of support.

Reply:
I updated to 4 months